src/services/octokit.ts

```
@@ -5,13 +5,16 @@ export async function fetchUserGitHubRepositories(session: Session) {
-  return {
-    data: githubRepos,
-  };
+  try {
+    const { data: githubRepos } = await octokit.repos.listForAuthenticatedUser({
+      per_page: 100,
+    });
+    return {
+      data: githubRepos,
+    };
+  } catch (error) {
+    throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)